modules/data-streams/src/test/java/org/elasticsearch/datastreams/action/GetDataStreamsResponseTests.java

```diff
@@ -43,22 +61,221 @@ protected Response mutateInstance(Response instance) {
+                Map<String, Object> dataStreamMap = (Map<String, Object>) dataStreams.get(0);
+                assertThat(dataStreamMap.get(DataStream.NAME_FIELD.getPreferredName()), is(dataStreamName));
+                // note that the prefer_ilm value is displayed at the top level even if the template backing the data stream doesn't have a
+                // policy specified anymore
+                assertThat(dataStreamMap.get(Response.DataStreamInfo.PREFER_ILM.getPreferredName()), is(false));
+                assertThat(dataStreamMap.get(Response.DataStreamInfo.ILM_POLICY_FIELD.getPreferredName()), is(nullValue()));
+                assertThat(dataStreamMap.get(Response.DataStreamInfo.LIFECYCLE_FIELD.getPreferredName()), is(Map.of("enabled", false)));
+                assertThat(
+                    dataStreamMap.get(Response.DataStreamInfo.NEXT_GENERATION_INDEX_MANAGED_BY.getPreferredName()),
+                    is(ManagedBy.UNMANAGED.displayValue)
+                );
+
+                List<Object> indicesRepresentation = (List<Object>) dataStreamMap.get(DataStream.INDICES_FIELD.getPreferredName());
+                Map<String, Object> firstGenIndexRepresentation = (Map<String, Object>) indicesRepresentation.get(0);
+                assertThat(firstGenIndexRepresentation.get("index_name"), is(firstGenerationIndex.getName()));
+                assertThat(firstGenIndexRepresentation.get(Response.DataStreamInfo.PREFER_ILM.getPreferredName()), is(true));
+                assertThat(firstGenIndexRepresentation.get(Response.DataStreamInfo.ILM_POLICY_FIELD.getPreferredName()), is(ilmPolicyName));
+                assertThat(
+                    firstGenIndexRepresentation.get(Response.DataStreamInfo.MANAGED_BY.getPreferredName()),
+                    is(ManagedBy.ILM.displayValue)
+                );
+
+                // the write index is managed by data stream lifecycle
+                Map<String, Object> writeIndexRepresentation = (Map<String, Object>) indicesRepresentation.get(2);
+                assertThat(writeIndexRepresentation.get("index_name"), is(writeIndex.getName()));
+                assertThat(writeIndexRepresentation.get(Response.DataStreamInfo.PREFER_ILM.getPreferredName()), is(false));
+                assertThat(writeIndexRepresentation.get(Response.DataStreamInfo.ILM_POLICY_FIELD.getPreferredName()), is(nullValue()));
+                assertThat(
+                    writeIndexRepresentation.get(Response.DataStreamInfo.MANAGED_BY.getPreferredName()),
+                    is(ManagedBy.UNMANAGED.displayValue)
+                );
+            }
+        }
+    }
+
+    public void testManagedByDisplayValuesDontAccidentalyChange() {
+        // UI might derive logic based on the display values so any changes should be coordinated with the UI team
+        assertThat(ManagedBy.ILM.displayValue, is("Index Lifecycle Management"));
+        assertThat(ManagedBy.LIFECYCLE.displayValue, is("Data stream lifecycle"));
```

Comment:
@alisonelizabeth @tylerperk @sabarasaba @mattc58 @gmarouli 

Note the UI copy wouldn't be camel cased for DSL - `Data stream lifecycle` (as opposed to the camel cased `Index Lifecycle Management`) . This was a choice such that we talk about it similarly to `Index settings`. 
If you disagree or have a better proposal let me know.